values/registry1-values.yaml

```diff
@@ -8,3 +8,33 @@ configLogicalBackup:
 # Note there is not a spilo image in registry1
 configGeneral:
   docker_image: "ghcr.io/zalando/spilo-15:3.2-p1"
+  sidecars:
+    - name: "exporter"
+      image: "registry1.dso.mil/ironbank/opensource/prometheus/postgres-exporter:v0.15.0"
```

Comment:
Can we move the sidecar configuration to the postgresql cr in the config chart? 

This would allow someone else to deploy a global config vs us reserving the key, would eliminate the need to define the whole block twice and would make it not hardcoded to only work against `postgres.pg-cluster` due to the hardcoded secret ref. 

The workflow for this would be:
1. move sidecar block to the the chart/templates/postgres-minimal.yaml
2. add a helm value template just for the image
3. create a registry1-config-value.yaml and upstream-config-values.yaml and set `image: <upstream_image>` and `image: <registry1_image>` in each.
4. in each of the flavor componenets. in the top level zarf yaml, add:

```
      - name: uds-postgres-config
        valuesFiles:
          - ./values/upstream-config-values.yaml
```
and its registry1 counterpart.

Reply:
see https://github.com/zalando/postgres-operator/blob/73f72414f65690fe38d6892559f085e9e1f3f33d/manifests/postgresql.crd.yaml#L473

Reply:
@zachariahmiller great solution, thanks so much!